src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -171,6 +171,7 @@ public void keyboardShortcutsTest() {
         assertEquals(repoSelectorComboBox.getValue(), "dummy/dummy");
         // test shortcut when focus is on panel
         PlatformEx.waitOnFxThread();
+        sleep(1000);
```

Comment:
I think it is. I can't remember clearly but I tried that before and it works, except that that time I used a while loop waiting for the node, and if the node is not available yet, I will sleep. I will try it out.